src/sentry/api/serializers/models/alert_rule.py

```diff
@@ -111,7 +111,7 @@ def get_attrs(
             )
             for action in serialized.get("actions", []):
                 install = sentry_app_installations_by_sentry_app_id.get(
-                    str(action.get("sentryAppId"))
+                    str(action.get("sentryAppId")), {}
```

Comment:
isn't the action none?

Reply:
I could break this into 2 lines, but if I make the default an empty dict then the chained `.get()` won't fail